js/bybit.js

```diff
@@ -3098,6 +3098,13 @@ module.exports = class bybit extends Exchange {
         if (price !== undefined) {
             request['p_r_price'] = this.priceToPrecision (symbol, price);
         }
+        if (isConditionalOrder) {
+            const triggerPrice = this.safeValueN (params, [ 'stopPrice', 'triggerPrice' ]);
```

Comment:
@sc0Vu we must support all three variations here:
- CCXT-specific triggerPrice (current approach)
- CCXT-specific stopPrice (backward-compatible approach)
- exchange-specific `p_r_trigger_price`
If any of those three are specified, we must format it to precision and send `p_r_trigger_price` to a corresponding conditional orders endpoint.